### PR TITLE
Add new testing Jenkins pipeline libraries

### PR DESCRIPTION
**Pull request by sondreso**

### CODE REVIEW

**Comment by sondreso** [comment]
Jenkins test this please

**Comment by sondreso** [comment]
jenkins test this please

**Comment by sondreso** [comment]
Jenkins test this please

**Comment by sondreso** [comment]
jenkins test this please

**Comment by sondreso** [comment]
jenkins test this please

**Comment by sondreso** [comment]
jenkins test this please

**Comment by sondreso** [comment]
jenkins test this please

**Comment by sondreso** [comment]
test this please

**Comment by sondreso** [comment]
test this please

**Comment by ManInFez** [comment]
Test this please!

File: ci/jenkins/testkomodo.sh [hunk 1]
```diff
@@ -1,75 +1,60 @@
-#!/bin/bash
-set -xe
-PROJECT=libres
-RELEASE_PATH=${KOMODO_ROOT}/${RELEASE_NAME}
-GIT=${SDPSOFT}/bin/git
-source $KOMODO_ROOT/$RELEASE_NAME/enable
-export LD_LIBRARY_PATH=${RELEASE_PATH}/root/lib:${RELEASE_PATH}/root/lib64
+set -eu
```
Comment (ManInFez) [inline ci/jenkins/testkomodo.sh]: no!

**Review by ManInFez — APPROVED** [review]
minor comment

**Pull request merged**